src/validators/duplicate_stops.rs

```diff
@@ -24,9 +24,9 @@ fn too_close_stops(stop_a: &gtfs_structures::Stop, stop_b: &gtfs_structures::Sto
     let a = Point::new(stop_a.longitude, stop_a.latitude);
     let b = Point::new(stop_b.longitude, stop_b.latitude);
     let min_distance = match stop_a.location_type {
-        gtfs_structures::LocationType::StopPoint => 12.,
```

Comment:
aren't those values a bit low ?

it's a problem if there are 2 stops with the same name within 10mn no ?